sigs/tfx-addons/CHARTER.md

```diff
@@ -20,15 +20,15 @@ These projects will begin as proposals to the SIG, and upon approval will be led
 Although TFX is an open-source project and we welcome contributions to TFX itself, **this SIG does not include contributions or additions to core TFX**.  It is focused only on building community-contributed and maintained additions on top of core TFX.  [Core TFX has its own repo](https://github.com/tensorflow/tfx), and PRs and issues will continue to be managed there. **In addition, all contributions must not violate the [Google AI Principles](https://ai.google/principles/) or [Responsible AI Practices](https://ai.google/responsibilities/responsible-ai-practices/).**
 
 ## Membership
-We encourage any developers working in production ML environments, infrastructure, or applications to join and participate in the activities of the SIG. Whether you are working on advancing the platform, prototyping or building specific applications, or authoring new components, templates, libraries, and/or orchestrator support, we welcome your feedback on and contributions to TFX and its tooling, and are eager to hear about any downstream results, implementations, and extensions. 
+We encourage any developers working in production ML environments, infrastructure, or applications to join and participate in the activities of the SIG. Whether you are working on advancing the platform, prototyping or building specific applications, or authoring new components, templates, libraries, and/or orchestrator support, we welcome your feedback on and contributions to TFX and its tooling, and are eager to hear about any downstream results, implementations, and extensions.
 
-We have multiple channels for participation, and publicly archive discussions in our user group mailing list: 
+We have multiple channels for participation, and publicly archive discussions in our user group mailing list:
 - tfx@tensorflow.org – our general mailing list that all are welcome to join (archive group: https://groups.google.com/a/tensorflow.org/g/tfx)
 - tfx-addons-core@tensorflow.org – Google group for core team (to be created)
```

Comment:
I think we can remove this and instead just reference the sig tfx-addons google group. No need to keep a separate group for core.

Reply:
Yeah, we haven't been using that much.  It does still exist, and we can use it, but it probably doesn't need to be mentioned here.